### DISCUSSION TITLE
SCA Scan setup

### DISCUSSION THREAD

**Discussion by RosadoJ** [wazuh/wazuh#25803] [General]
Does anyone know how I can fix the SCA Scan for a monitored Ubuntu PC, I checked the config files, and it's set to enable. Any ideas?
![Screenshot from 2024-09-19 21-44-09](https://github.com/user-attachments/assets/dce0dfc4-c0e9-43c4-a4c3-ecfbb30bd0e2)

**Comment by schoonersoldier** [comment]
you need to enable the module witihn the server ossec.conf file and also check whether anything needs enabling on the agent you want to use the module with.
